### DISCUSSION TITLE
message:"error getting response from" Error 500

### DISCUSSION THREAD

**Discussion by felipegimenez** [grafana/grafana-infinity-datasource#137] [Q&A]
Hello, @yesoreyeram!

First, congratulations for your work with grafana-infinity-datasource. It's a very nice plugin.

I tried use the plugin, but i got this error in Grafana Dashboard:

```
error getting response from https://jsonplaceholder.typicode.com/users
Object
status:500
statusText:"Internal Server Error"
data:Object
message:"error getting response from https://jsonplaceholder.typicode.com/users
"
error:"Internal Server Error"
response:"error getting response from https://jsonplaceholder.typicode.com/users
"
config:Object
method:"POST"
url:"api/datasources/110/resources/proxy"
data:Object
retry:0
headers:Object
hideFromInspector:true
message:"error getting response from https://jsonplaceholder.typicode.com/users

```
Log of application:
`lvl=eror msg="Request Completed" logger=context = orgId=1 method=POST path=/api/datasources/210/resources/proxy status=500`

Can you help me?

**Comment by yesoreyeram** [comment]
Have you tried creating a new instance of data source config (without URL in the config)?

**Reply by felipegimenez** [reply]
Yes, but i caught the same error. 🙁

**Comment by gk7279** [comment]
Same issue with me as well when using docker image of grafana.

**Comment by mengyuana** [comment]
got the same issue, is there any solution?

**Comment by tmuhamma** [comment]
Any fix?

**Comment by KravchenkoAlex** [comment]
I just relaunch grafana, and it works, but i'm not sure, that this is correct behavior how to fix it
